### DISCUSSION TITLE
Upgrading to compute module 4 or 5

### DISCUSSION THREAD

**Discussion by NeilDuToit92** [micro-nova/AmpliPi#635] [General]
Are there any plans to replace the current Pi CM3+ module with a CM4 or CM5 module?

Seeing as the CM3 is getting a bit older and the CM5 is being released this year, are there any plans to update the project to be compatible with the reduced GPIO available on the newer modules?

**Comment by linknum23** [comment]
This is something we are considering, however we don't have any concrete plans to announce yet.
